src/main/java/com/thoughtworks/gauge/execution/parameters/ParsingException.java

```diff
@@ -0,0 +1,17 @@
+package com.thoughtworks.gauge.execution.parameters;
+
+import gauge.messages.Spec.ProtoExecutionResult;
+
+public class ParsingException extends Exception {
+    private static final long serialVersionUID = -4205182693938362914L;

```

Comment:
We are not serializing/deserializing this exception. Is this field really required?